File: codigos/1 - estrutura sequencial/1009 - Salario com Bonus/Main.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift();};
var A = prompt("Digite o seu nome");
var B = parseFloat(prompt("Digite o seu salario"));
var C = parseFloat(prompt("Digite o montante total das vendas efetuadas por voce"));
var D = (B+0.15*C).toFixed(2);
console.log("TOTAL = R$ "+D);

File: codigos/1 - estrutura sequencial/1009 - Salario com Bonus/Main.clj
(defn main []
    (read)
    (let [salario-fixo (double (read))
          receita-vendas (double (read))
          salario-total (+ salario-fixo (* 0.15 receita-vendas))]
        (println "TOTAL = R$" (format "%.2f" salario-total))))

(main)